content/docs/intro/concepts/programming-model.md

```diff
@@ -56,13 +56,26 @@ if err != nil {
 }
 ```
 
+```csharp
+using Pulumi;
+using Pulumi.Aws.Ec2;
+
+// Inside call to Deployment.Run
+var group = new SecurityGroup(...);
+var server = new Instance("webserver-www", new InstanceArgs
+{
+    //...
+    SecurityGroups = { group.Name }, // reference the security group resource above
+});
+```
+
 To publish values that you wish to access outside your application, create a [stack output](#stack-outputs) via module exports.
 
 In Pulumi, you can group multiple resources in a [component](#components). A component is a logical container for physical cloud resources and affects how resources are grouped in the CLI and the pulumi.com console.
 
 ## Programs {#programs}
 
-Pulumi programs are authored in general-purpose programming languages such as [JavaScript]({{< relref "/docs/intro/languages/javascript" >}}) or [Python]({{< relref "/docs/intro/languages/python" >}}). You can use any packages supported by the language's package manager, as well as [Pulumi packages]({{< relref "/docs/reference/pkg" >}}).
+Pulumi programs are authored in general-purpose programming languages such as [JavaScript]({{< relref "/docs/intro/languages/javascript" >}}) or [Python]({{< relref "/docs/intro/languages/python" >}}) or a .NET Language (in preview). You can use any packages supported by the language's package manager, as well as [Pulumi packages]({{< relref "/docs/reference/pkg" >}}).
```

Comment:
Should the .NET bit link to the /docs/intro/languages/dotnet page, like the other languages do?

Reply:
We don't have that page yet.  So i can't link to it without breaks.